modules/augment/include/opencv2/augment/transform.hpp

```diff
@@ -7,75 +7,66 @@
 #include <opencv2/core.hpp>
 #include <vector>
 
-namespace cv {
-namespace augment {
-    //! @addtogroup augment
-        //! @{
+namespace cv { namespace augment {
 
+class CV_EXPORTS_W Transform
+{
+public:
+    /* @brief Constructor
+    */
+    CV_WRAP Transform();
 
-    class CV_EXPORTS_W Transform
-    {
-    public:
-        /* @brief Constructor
-            @param _probability probability that this transformation will be applied to each image
-        */
-        CV_WRAP Transform();
+    /* @brief Destructor
+    */
+    virtual ~Transform();
 
-        /* @brief Destructor
-        */
-        virtual ~Transform();
+    /* @brief Apply the transformation to a single image (this is overridden by transformations implementation)
+        @param _src Input image to be tranformed
+        @param _dst Output (transformed) image
+    */
+    CV_WRAP virtual void image(InputArray _src, OutputArray _dst);
 
-        /* @brief Apply the transformation to a single image (this is overridden by transformations implementation)
-            @param _src Input image to be tranformed
-            @param _dst Output (transformed) image
-        */
-        CV_WRAP virtual void image(InputArray _src, OutputArray _dst);
+    /* @brief Apply the transformation for a single point (this is overridden by transformations implementation)
+        @param src Input point to be tranformed
+    */
+    virtual Point2f point(const Point2f& src);
 
-        /* @brief Apply the transformation for a single point (this is overridden by transformations implementation)
-           @param image the image that has the point to be transformed
-           @param src Input point to be tranformed
-        */
-        virtual Point2f point(InputArray image,const Point2f& src);
 
+    /* @brief Apply the transformation for a rectangle
+        @param box Vec4f consisting of (x1, y1, x2, y1) corresponding to (top left, bottom right)
+    */
+    virtual Vec4f rectangle(const Vec4f& src);
```

Comment:
Please use cv::Rect2f instead.

Reply:
Rect2f uses xmin,ymin,w,h while I am using xmin,ymin,xmax,ymax notation which is more popular in deep learning data sets.. that's why I had to change it